src/meta/WebAnimation.ts

```diff
@@ -113,7 +117,12 @@ export class WebAnimations extends Base {
 		}
 	}
 
-	animate(key: string, animateProperties: AnimationProperties | AnimationProperties[]) {
+	animate(
+		key: string,
+		animateProperties:
+			| AnimationPropertiesOrAnimationPropertiesFunction
```

Comment:
The leading pipe surprised me but them Tom pointed out it was this! https://github.com/prettier/prettier/issues/2069